core/src/core-api/index.js

```diff
@@ -13,6 +14,7 @@ export const LuigiNavigation = navigation;
 export const LuigiI18N = i18n;
 export const LuigiCustomMessages = customMessages;
 export const LuigiUX = ux;
+export const LuigiTheming = theming;
```

Comment:
Theming is a very broad acronym. If this is only related to color (no fonts, sizes, icons) then what about renaming it to LuigiColorTeming?